content/en/docs/components/katib/katib-config.md

```diff
@@ -0,0 +1,211 @@
++++
+title = "Katib Configuration Overview"
+description = "How to make changes in Katib configuration"
+weight = 40
+                    
++++
+
+This page describes information about
+[Katib config](https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/katib-controller/katib-config.yaml).
+
+Katib config is the Kubernetes
+[Config Map](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/) that contains information about:
+
+1. Current [metrics collectors](/docs/components/katib/experiment/#metrics-collector) (`key = metrics-collector-sidecar`)
+1. Current [algorithms](/docs/components/katib/experiment/#search-algorithms-in-detail) (suggestions) (`key = suggestion`).
+
+Katib Config Map must be deployed in
+[`KATIB_CORE_NAMESPACE`](/docs/components/katib/env-variables/#katib-controller)
+namespace with `katib-config` name. Katib controller parses Katib config when you submit Experiment.
+
+You can edit this Config Map even after deploying Katib.
+
+If you deploy Katib in Kubeflow namespace, to edit Katib config run this:
```

Comment:
Maybe just change the order here, use a different (present continuous) tense, and "the" (the definite article) (I checked with the K8s [guide](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/), as I'm not 100% sure about these):

```suggestion
If you are deploying Katib in the Kubeflow namespace, run this command to edit your Katib config:
```

Reply:
Agree, changed.